lambdas/tests/unit/services/test_update_upload_state_service.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+from botocore.exceptions import ClientError
+from services.update_upload_state_service import UpdateUploadStateService
+from tests.unit.helpers.data.update_upload_state import (
+    MOCK_ALL_DOCUMENTS_REQUEST,
+    MOCK_ARF_DOCTYPE,
+    MOCK_ARF_DOCUMENTS_REQUEST,
+    MOCK_DOCUMENT_REFERENCE,
+    MOCK_LG_DOCTYPE,
+    MOCK_LG_DOCUMENTS_REQUEST,
+    MOCK_NO_DOCTYPE_REQUEST,
+    MOCK_NO_FIELDS_REQUEST,
+    MOCK_NO_FILES_REQUEST,
+    MOCK_NO_REFERENCE_REQUEST,
+)
+from utils.lambda_exceptions import UpdateUploadStateException
+
+
+@pytest.fixture
+def patched_service(set_env, mocker):
+    service = UpdateUploadStateService()
+    mock_dynamo_service = mocker.patch.object(service, "dynamo_service")
+    mocker.patch.object(mock_dynamo_service, "update_item")
+    yield service
+
+
+@pytest.fixture
+def mock_update_document(patched_service, mocker):
+    yield mocker.patch.object(patched_service, "update_document")
```

Comment:
Could we add some tests for `update_document` and `format_update` as they've both been patched and not tested